test cases/windows/17 cpp17/meson.build

```diff
@@ -1,8 +1,8 @@
+project('msvc_cpp17', 'cpp', default_options: ['cpp_std=c++17'])
+
 if meson.get_compiler('cpp').version().version_compare('<19.11')
```

Comment:
you need to check that the compiler is msvc before checking the version, this could be gcc, clang, icc, or a host of other compilers.

Also, please squash this back into the previous patch.

Reply:
will do, I planned to make sure it worked first.